config/kibana.yml

```diff
@@ -1,98 +1,98 @@
+#elasticsearch.pingTimeout: 1500
 
-# Time in milliseconds to wait for responses from the back end or Elasticsearch. This value
-# must be a positive integer.
-# elasticsearch.requestTimeout: 30000
+#Time in milliseconds to wait for responses from the back end or Elasticsearch. This value
+#must be a positive integer.
+#elasticsearch.requestTimeout: 30000
 
-# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
-# headers, set this value to [] (an empty list).
-# elasticsearch.requestHeadersWhitelist: [ authorization ]
+#List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+#headers, set this value to [] (an empty list).
+#elasticsearch.requestHeadersWhitelist: [ authorization ]
 
-# Header names and values that are sent to Elasticsearch. Any custom headers cannot be overwritten
-# by client-side headers, regardless of the elasticsearch.requestHeadersWhitelist configuration.
-# elasticsearch.customHeaders: {}
+#Header names and values that are sent to Elasticsearch. Any custom headers cannot be overwritten
+#by client-side headers, regardless of the elasticsearch.requestHeadersWhitelist configuration.
+#elasticsearch.customHeaders: {}
 
-# Time in milliseconds for Elasticsearch to wait for responses from shards. Set to 0 to disable.
-# elasticsearch.shardTimeout: 0
+#Time in milliseconds for Elasticsearch to wait for responses from shards. Set to 0 to disable.
+#elasticsearch.shardTimeout: 0
 
-# Time in milliseconds to wait for Elasticsearch at Kibana startup before retrying.
-# elasticsearch.startupTimeout: 5000
+#Time in milliseconds to wait for Elasticsearch at Kibana startup before retrying.
+#elasticsearch.startupTimeout: 5000
 
-# Specifies the path where Kibana creates the process ID file.
-# pid.file: /var/run/kibana.pid
+#Specifies the path where Kibana creates the process ID file.
+#pid.file: /var/run/kibana.pid
 
-# Enables you specify a file where Kibana stores log output.
-# logging.dest: stdout
+#Enables you specify a file where Kibana stores log output.
+#logging.dest: stdout
 
-# Set the value of this setting to true to suppress all logging output.
-# logging.silent: false
+#Set the value of this setting to true to suppress all logging output.
+#logging.silent: false
 
-# Set the value of this setting to true to suppress all logging output other than error messages.
-# logging.quiet: false
+#Set the value of this setting to true to suppress all logging output other than error messages.
+#logging.quiet: false
 
-# Set the value of this setting to true to log all events, including system usage information
-# and all requests.
-# logging.verbose: false
+#Set the value of this setting to true to log all events, including system usage information
+#and all requests.
+#logging.verbose: false
 
-# Set the interval in milliseconds to sample system and process performance
-# metrics. Minimum is 100ms. Defaults to 5000.
-# ops.interval: 5000
```

Comment:
Awesome, thanks for submitting this one! Can I suggest you maintain the spaces before the non-config sections, ie the actual descriptive comments, and just remove them for the config options. Like so;

```
# Kibana is served by a back end server. This setting specifies the port to use.
#server.port: 5601
```

Take a look at https://github.com/elastic/elasticsearch/pull/20094/files, which is the Elasticsearch version of the same thing :)